llvm/lib/Target/LoongArch/LoongArchInstrInfo.td

```diff
@@ -854,6 +854,12 @@ def BREAK   : MISC_I15<0x002a0000>;
 def RDTIMEL_W : RDTIME_2R<0x00006000>;
 def RDTIMEH_W : RDTIME_2R<0x00006400>;
 
+// The CPUCFG instruction offers a reliable way to probing CPU features.
+// Although it's undefined on LA32R, adding compiler support enables the
```

Comment:
Maybe this is better: "Although support is not guaranteed on LA32R, having compiler support nevertheless enables ..."

(Apparently it's not forbidden to have it included in a LA32R implementation, it's just not being in the mandatory set.)

Reply:
Looks good to me. :smiley: